cmd/lightclient/sentinel/state.go

```diff
@@ -0,0 +1,213 @@
+package sentinel
```

Comment:
Move this out of sentinel, sentinel is strictly for p2p, create a new go package named `lightclient` inside the `lightclient` in a new `lightclient` folder